### ISSUE TITLE
问题交流

### ISSUE THREAD

**Issue by zhengpengfeichn** [zhengpengfeichn/VxTerm#2]
在使用过程中，如发现问题，或功能不足，需要交流使用心得，可以在此留下您宝贵的信息。
我一定会积极的响应。
谢谢。

**Comment by peafowl-wang** [comment]
不能用密钥，不是密码的登陆不了，谢谢

**Comment by zhengpengfeichn** [comment]
是的。由于本人长期工作中并未使用密钥登录的场景，对此场景比较陌生。本人接下来会尽量的查询关于使用密钥的相关信息，对它进行优化。
期待您的继续支持。感谢。

**Comment by yinkaisheng** [comment]
@zhengpengfeichn sftp界面文件列表字体有点太小，希望增加选项可以设置 文件列表字体大小，关闭sftp窗口后再打开，希望可以记住之前的sftp窗口大小。
